packages/monitor-v2/src/monitor-polymarket/MonitorProposalsOrderBook.ts

```diff
@@ -54,7 +54,14 @@ export async function monitorTransactionsProposedOrderBook(
   const marketsWithEventData: PolymarketWithEventData[] = marketsWithAncillary
     .filter((market) => proposalEvents.find((event) => event.ancillaryData === market.ancillaryData))
     .map((market) => {
-      const event = proposalEvents.find((event) => event.ancillaryData === market.ancillaryData);
+      const events = proposalEvents.filter((event) => event.ancillaryData === market.ancillaryData);
```

Comment:
One separate question (does not need to be resolved in this PR): what would happen if there were two markets with identical ancillary data values?

Reply:
@mrice32 good point but that could be a followup PR next week, I think we could use the `requestTimestamp` available in the polymarket adapters tu uniquely identify them
<img width="1377" alt="Screenshot 2023-05-12 at 20 27 45" src="https://github.com/UMAprotocol/protocol/assets/53186719/a6182fcf-1de4-452d-834a-91b088511993">